src/modules/75-cf/components/FlagActivation/FlagActivation.tsx

```diff
@@ -483,7 +483,7 @@ const FlagActivation: React.FC<FlagActivationProps> = props => {
       })}
       onSubmit={onSaveChanges}
     >
-      {formikProps => {
+      {(formikProps: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
      {(formikProps: unknown) => {
```